src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -250,7 +233,8 @@ internal override void RemoveExecutionPolicy(PropertyScope scope, string shellId
             // Defaults to system wide.
             if (PropertyScope.CurrentUser == scope)
             {
-                scopeDirectory = GetCurrentUserConfigDirectory();
+                CreateUserConfigDirectory();
```

Comment:
Minor question: is it really necessary to create the file when it's requested to remove a property? If the file doesn't exist, then no action needed, and the removal will be sure successful.